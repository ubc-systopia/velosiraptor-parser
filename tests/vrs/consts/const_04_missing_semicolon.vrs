const FOO: int = 1234
const FOO: int = 1234 asdf;
