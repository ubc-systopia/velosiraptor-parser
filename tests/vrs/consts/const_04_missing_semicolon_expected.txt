error: expected `;`, found `const`
     --> tests/vrs/consts/const_04_missing_semicolon.vrs:2:1
      |
    2 |  const FOO: int = 1234 asdf;
      |  ^^^^^ help: add `;` here.
